index.html

```diff
@@ -52,11 +52,40 @@
       #counter.fail
       {
         color:red;
+        box-shadow:0px 0px 100px 0px #600;
       }
     </style>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
     <script>
-      // Put some code in here!
+      $(document).ready(start);
+
+      function start(){
+        var TIME_START = 1234;
+        var time = TIME_START;
+        var bActive = false;
+        var counter = $("#counter");
+
+        counter.click(function(e){
+          e.preventDefault();
+          if(bActive){bActive=false;}
+          else{
+            if(counter.hasClass("fail")){counter.toggleClass("fail");}
+            if(time > 0){bActive=true;}
+            time=TIME_START;
+          }
+        })
+
+        setInterval(update,1);
+
+        function update(e){
+          counter.prop("value",time);
+          if(time == 0 && !counter.hasClass("fail")){counter.toggleClass("fail")}
+          if(bActive){
+            if(time > 0){time-=1;}else{bActive = false;}
```

Comment:
Thanks for keeping this simple!

This line looks a little gnarly. Different strokes, but I prefer:

```js
if(time > 0) time -= 1;
else bActive = false;
```